Add mapping contest official support article (#8643)
ppy/osu-wiki: wiki/Contests/Official_support/en.md

---
tags:
  - badge
  - badges
  - badged
---

# Official contest support

> [Beatmapping contests](/wiki/Contests) are eligible to recieve support from the [osu! team](/wiki/People/The_Team) provided they meet certain criteria.

Reviewer: The title of this article is for contests in general, so limiting the article to just beatmap contests feels weird. I know that other types of contests are very rare, however a note about these should be left here. As someone who hosts other kinds of contests myself I know how terrible the wiki already is at guiding people like me, so this would be a good opportunity to address that.

*For tournament support, see: [Official tournament support](/wiki/Tournaments/Official_support).*

## Benefits

Officially supported contests have access to certain benefits:

- **Profile badge prizes for first-place winners.** See profile badge design criteria on the [official tournament support article](/wiki/Tournaments/Official_support#profile-badges).
- **Progress towards *Elite Mapper* user title.** See [contest points](/wiki/Contests/Contest_points) for more details.

## Eligibility

Contests which abide by the following criteria are eligible for official support for **profile badges**:

- **The contest must meet either of these criteria:**
  - The contest is organised and run by an experienced team of accomplished and reputable staff, as decided by the osu! team.
  - The contest has successfully run for one iteration without official support.
- **The contest must have a clearly defined set of rules.**

> - **The contest must accept submissions for only one game mode.** Most users specialize in one game mode, so it's not reliable to evaluate mapping quality across multiple game modes.

Reviewer 2: ```suggestion
- **The contest must accept submissions for only one game mode.** Most users specialise in one game mode, so it is not reliable to evaluate mapping quality across multiple game modes.
```

- **The contest must have at least 10 submissions that adhere to the contest's rules.** Contests with fewer valid submissions are not deemed competitive enough.

> - **The contest must have a clearly defined judging criteria based on mapping merit.** These ensure a contest follows an understandable quality standard in line with other mapping contests.

Reviewer 2: does this mean badges cannot be received from winning public votings?

- **The contest's results must be determined by valuing scores from all judges equally.** Using standard deviation to determine final scores is preferred but not required.

> - **The contest's results must be publicly displayed with judge scores/comments visible.** The judges may be anonymized publicly, but the osu! team may ask for un-anonymized judge details if necessary.

Reviewer 2: ```suggestion
- **The contest's results must be publicly displayed with judge scores/comments visible.** The judges may be anonymised publicly, but the osu! team may ask for un-anonymised judge details if necessary.
```

- **The contest must have at least 5 judges.** Contests with fewer judges allow each judge too much influence on overall results.
- **A user cannot participate in more than one submission to the contest.**

> - **Submissions to the contest should be anonymized.** This limits judging bias removing name association between a mapper and their map. If a contest has special circumstances that warrant removing anonymity, this rule may be lifted, but otherwise must be followed.

Reviewer 2: ```suggestion
- **Submissions to the contest should be anonymised.** This limits judging bias removing name association between a mapper and their map. This rule may be ignored only if a contest has special circumstances that warrant removing anonymity.
```

> - **The contest's organisers must not participate in the contest.** This includes both submitting a map *and* judging submissions. Anyone with access to judging information or un-anonymized submissions is considered a "contest organiser".

Reviewer 2: ```suggestion
- **The contest's organisers must not participate in the contest.** This includes both submitting a map *and* judging submissions. Anyone with access to judging information or un-anonymised submissions is considered a "contest organiser".
```

In addition to the criteria above, contests which abide by these rules are eligible for official support for **[contest points](/wiki/Contests/Contest_points) to earn the *Elite Mapper* user title:**

> - **The contest must exclusively use songs listed in osu!'s [Featured Artist library](https://osu.ppy.sh/beatmaps/artists).**

Reviewer 3: feels like this should be expanded to not only limit contest to songs that are featured in the fa listing tbh :(

recently there have been contests such as [this](http://osu.ppy.sh/community/forums/topics/1621520) and [this](http://osu.ppy.sh/community/forums/topics/1596151) that features a custom-made song for the contest

Reviewer 2: so by extension, fa + songs with permission for use, or just fa + originals? mentioning just in case

> - **The contest must be published on [osu!'s front page news](https://osu.ppy.sh/home/news) and the [contest listing](https://osu.ppy.sh/community/contests).** For credibility of the *Elite Mapper* title, a contest must have maximum visibility.

Reviewer 2: does this entail only an initial announcement or a set of posts like announcement (and should registration already be opened/determined etc?) and results (+ voting open etc?)?

Reviewer 4: from experience I think it should be initial announcement + results announcement, would be nice to clarify that here if that's still the case

Author: i think the initial announcement is what's most important, since it's what increases visibility. results are good to have too ofc, but other posts aren't always part of contests

## Requesting support

> If your contest satisfies the [eligibility criteria](#eligibility), you can make a request for support by sending an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh) with the following information:

Reviewer 2: the contest must have concluded right?

Author: the criteria requires the contest to be completed already

- A brief description of your contest, including any applicable dates, game modes, and any other appropriate features
- A list of all organisers and staff involved in the contest and their respective roles
- Any links to:
  - The contest's results
  - The forum thread for your contest.
  - Public Discord servers or any other off-site chatrooms used for the contest.
  - Previous iterations of the same contest series (if applicable).
  - Your contest on the [Mappers' Guild contest listing](https://mappersguild.com/contests/listing) (if applicable).

Once we receive your request, the osu! team will send an approval or denial of your contest's official support status. 

> The osu! team may choose to deny official support to any contest that fits the eligibility criteria at their discretion. This ensures no foul play occurs by contest organisers taking advantage of criteria loopholes, so if you're running a contest without ill intent, you likely don't need to worry about this.

Reviewer 2: ```suggestion
The osu! team may choose to deny official support to any contest that fits the eligibility criteria at their discretion. This ensures no foul play occurs by contest organisers taking advantage of criteria loopholes, so if you are running a contest without ill intent, you likely do not need to worry about this.
```
